letshelp-letsencrypt/letshelp_letsencrypt/letshelp_letsencrypt_apache.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python
```

Comment:
Please rename this file to just `apache.py` and set a preferred installed script name through `setup.py` entry points (see one of my previous comments).